site/public/js/docker_interface.js

```diff
@@ -0,0 +1,63 @@
+
+function checkDockerInterfaceJobs(){
```

Comment:
fix the indentation in this file to be just spaces, with each indentation level at 2 spaces